### ISSUE TITLE
Account recovery request

### ISSUE THREAD

**Issue by ashahba** [pypi/support#3617]
### PyPI Username

IntelAI

### Reason for request

Lost 2FA authentication app or security token

I have the associated email address but 2FA went into action when I was not prepared for it.
Now nothing works and I can't login to my account. I have tried with password, Recovery tokens but still no success.

Please help.

Thanks.

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

### Acknowledgement

- [X] I understand that it may take a significant amount of time to process my account recovery request.

**Comment by ashahba** [comment]
@miketheman I really appreciate your help with this issue.
Thanks.

**Comment by ashahba** [comment]
@ewdurbin would you please able to help me recover my account?

Thanks.

**Comment by ewdurbin** [comment]
User `IntelAI` has valid recovery codes generated as of 2024-01-03 23:41:16.608523.

Please use those recovery codes to regain access to your account.

**Comment by ashahba** [comment]
> User `IntelAI` has valid recovery codes generated as of 2024-01-03 23:41:16.608523.
> 
> Please use those recovery codes to regain access to your account.

Thanks @ewdurbin 
Unfortunately, I've lost those codes and I need to generate new ones.

**Comment by ashahba** [comment]
Hi @ewdurbin  and @miketheman 
I see `account-recovery` issues files as early as 4 days ago have been resolved:
https://github.com/pypi/support/issues?q=is%3Aissue+is%3Aclosed+label%3Aaccount-recovery

This issue which deals with exactly same thing, have been opened for 3 weeks now!
Any comment as to what I'm missing here that you guys are not willing to unblock me?

All I need is a password reset so that I can regain access to my account.

Thanks.

**Comment by ashahba** [comment]
@di and @cmaureir would you guys be able to look into this?
My team at [`Intel`](https://github.com/IntelAI) relies on this account for business continuity.

I really appreciate your continues support you've provided throughout the years.

Thanks.

**Comment by ashahba** [comment]
Please keep in mind that the user `IntelAI` is the admin account who's currently locked out.
This user owns all these projects:
https://pypi.org/user/IntelAI/ and hopes to create more projects for team going forward.

individual projects have maintainers assigned but any admin related task like add or remove maintainers is blocked due to this issue.

**Comment by ashahba** [comment]
Hi,
Looking here: https://github.com/orgs/pypi/projects/2/views/1
My ticket is categorized under `Valid Recovery Codes`.
What does that mean? As I mentioned before I have lost those code they are no longer considered valid.

Thanks.

**Comment by ashahba** [comment]
Hi @ewdurbin none of 8 recovery codes work so they need to be regenerated.
I really appreciate it If you please help me regenerate new codes.

Thanks.

**Comment by Thespi-Brain** [comment]
Duplicate of #4433
